README.md

```diff
@@ -211,34 +219,88 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
 ```
 
+# How to get an output
+
+There are 4 possible outcomes depending on the input:
+
+1. **Input:** A valid payment file.
```

Comment:
Add the `request-editor` structure here were you havva topic sentence to specify what the output is, i.e. **To get a request awaiting debt data** (i.e. payment request for an amount less than the previously settled amount)

Also, for some reason your newlines aren't coming across here 🤔